eng/pipelines/coreclr/templates/build-job.yml

```diff
@@ -287,6 +287,15 @@ jobs:
             ${{ else }}:
               hostArchType: x64
 
+    - ${{ if and(in(parameters.osGroup, 'windows'), eq(parameters.archType, 'x86'), ne(parameters.compilerName, 'gcc'), ne(parameters.testGroup, 'clrTools'), eq(parameters.pgoType, '')) }}:
+        - template: /eng/pipelines/coreclr/templates/crossdac-build.yml
+          parameters:
+            archType: arm
+            osGroup: ${{ parameters.osGroup }}
+            osSubgroup: ${{ parameters.osSubgroup }}
+            isOfficialBuild: ${{ parameters.signBinaries }}
+            hostArchType: x86
```

Comment:
@hoyosjs Like this?

Reply:
I was just looking, and this part will fail: 

https://github.com/dotnet/runtime/blob/3bd383212013439c27097339031e48091ac74a3a/eng/pipelines/coreclr/templates/crossdac-build.yml#LL49C51-L49C78

That gets defined in

https://github.com/dotnet/runtime/blob/main/eng/pipelines/coreclr/templates/xplat-pipeline-job.yml#L85

This is the only place that uses these variables. So lets just hardcode them in with what the parameters expand to inline. It will be a lot more localized.

Reply:
Thanks for pushing the fix.